flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionVertexDeploymentTest.java

```diff
@@ -273,15 +270,7 @@ public void testFailExternallyDuringDeploy() {
 				}
 			};
 
-			TestingLogicalSlot testingLogicalSlot =
-				new TestingLogicalSlot(
-					new LocalTaskManagerLocation(),
-					blockSubmitGateway,
-					0,
-					new AllocationID(),
-					new SlotRequestId(),
-					new SlotSharingGroupId(),
-					null);
+			TestingLogicalSlot testingLogicalSlot = new TestingLogicalSlot(blockSubmitGateway);
```

Comment:
this change doesn't belong into this PR.

Reply:
This change belongs to commit `[hotfix] Introduce a convenient constructor of TestingLogicalSlot`. Does it make sense or I should revert it?

Reply:
hmm................................since we're updating the test anyway in another PR I'd move this change into that PR. btw, it might be a good idea to change the first constructor (without any arguments) to call the new one, to ensure they use the same defaults.